src/components/text-editor/text-editor.tsx

```diff
@@ -34,6 +38,12 @@ import { buildDefaultMenu } from './menu/default-menu';
     styleUrl: 'text-editor.scss',
 })
 export class TextEditor {
+    /**
+     * Additional menu items to be displayed in the editor toolbar
+     */
+    @Prop()
+    public additionalMenuItems: EditorPluginButton[];
+
```

Comment:
As a compromise for now, could we remove this prop and keep the rest? In this case we are not exposing any capability to alter the menu but we keep the lighter menu compared to prosemirrors basic example. What do you say @adrianschmidt ?

Reply:
Works for me 👍